NOTICE.md

```diff
@@ -14,6 +14,14 @@ file in the Carbyne Stack
 - Schoelzel Hanna [hanna.schoelzel@bosch.io](mailto:hanna.schoelzel@bosch.io)
 - Suschke Volker [volker.suschke@bosch.io](mailto:volker.suschke@bosch.io)
 
+### Honda Research Institute Europe GmbH
+
+- Kalman Graffi [Kalman.Graffi@honda-ri.de](mailto:Kalman.Graffi@honda-ri.de)
```

Comment:
Format should be `<lastname>` `<firstname>`. Please check lexical order again after adjusting.

Reply:
Since we have both contributors from synyx on behalf of HRI and ones from HRI, should the ones from HRI be first, or ordered lexically?

Reply:
As they are under the same "heading", i.e., HRI, I would say lexically.